projects/1.18/assets/compressedcreativity/pneumaticcraft/patchouli_books/book/zh_cn/entries/industrial_air_blower.json

```
@@ -1,21 +1,21 @@
 {
-    "name": "工业吹风机",
-    "icon": "compressedcreativity:industrial_air_blower",
-    "category": "pneumaticcraft:machines",
-    "advancement": "pneumaticcraft:pressure_tube",
-    "pages": [
-      {
-        "type": "text",
-        "text": "$(item)工业吹风机$()是更为高级的吹风机。$(br)它能承受更高的压力，加工速度也可更快。但同时会导致高压气体消耗增加，气流的吹拂距离也较短。$(br2)它的另一功能在于，当达到某温度时，其产生的气流可用于烟熏和熔炼。"
-      },
-      {
-        "type": "text",
-        "text": "可以安装各种$(l:pneumaticcraft:industrial_air_blower_meshes)纱网$()以改变其功用。$(br2)默认状况下，$(item)工业吹风机$()所产生的气流在温度超过100℃时可用于烟熏，超过300℃时则可用于熔炼。"
-      },
-      {
-        "type": "crafting",
-        "recipe": "compressedcreativity:crafting/industrial_air_blower",
-        "text": "$(item)工业吹风机$()$(br2)的合成表"

```

```suggestion
      "text": "$(item)工业吹风机$()的合成表$(br2)"
```
怎么感觉这patchouli标记错位了（

---

br2相当于换两行
不知道作者为啥留着这个东西（）